app/client/components/MMAPage.tsx

```diff
@@ -98,6 +101,8 @@ const MMARouter = () => (
         <AccountOverview path="/" />
         <Billing path="/billing" />
 
+        <CancelReminders path="/cancel-reminders" />
```

Comment:
Could you please put this at the bottom of the Router after at a `// Does not require sign in` comment?
It will help anything that looks at it in the future better understand that this page doesn't require sign in.

Reply:
👍 done